CONTRIBUTING.md

```diff
@@ -0,0 +1,29 @@
+# Contributing to Estuary
+
+Estuary welcomes contributions to our [open source projects on Github](https://github.com/application-research/estuary).
+
+## Issues
+
+Feel free to submit issues and enhancement requests.
+
+Please use [Estuary Support](https://docs.estuary.tech/feedback) to report specific bugs and errors.
+
+# Contributing
+------------
+
+Please refer to each project's style and contribution guidelines for submitting patches and additions. In general, we follow the "fork-and-pull" Git workflow.
+
+ 1. **Fork** the repo on GitHub
+ 2. **Clone** the project to your own machine
+ 3. **Commit** changes to your own branch
+ 4. **Push** your work back up to your fork
+ 5. Submit a **Pull request** so that we can review your changes
+
+NOTE: Be sure to merge the latest from "upstream" before making a pull request!
+
+## Copyright and Licensing
+-----------------------
+
+Most Estuary open source projects are licensed under the [Apache 2.0 license](http://www.apache.org/licenses/LICENSE-2.0).
+
+To view the license, go [here](LICENSE.md)
```

Comment:
I personally think more is less. So maybe just remove this section altogether. In terms of repo design (if that's even a thing) it's pretty well-known that `LICENSE.md` holds license information

Reply:
Resolved on the latest commit.